waspc/cli/src/Wasp/Cli/Command/Db/Start.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Wasp.Cli.Command.Db.Start
+  ( start,
+  )
+where
+
+import Control.Monad.IO.Class (liftIO)
+import StrongPath (Abs, Dir, Dir', Path', Rel, fromAbsDir, reldir, (</>))
+import System.Process (callCommand)
+import qualified Wasp.AppSpec as AS
+import qualified Wasp.AppSpec.App as AS.App
+import qualified Wasp.AppSpec.App.Db as AS.App.Db
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.Common
+  ( findWaspProjectRootDirFromCwd,
+    parseWaspFileC,
+  )
+import Wasp.Cli.Command.Message (cliSendMessageC)
+import Wasp.Cli.Common (WaspProjectDir, dotWaspDirInWaspProjectDir)
+import qualified Wasp.Db.Postgres as P
+import qualified Wasp.Message as Msg
+
+start :: Command ()
+start = do
+  waspProjectDir <- findWaspProjectRootDirFromCwd
+  decls <- parseWaspFileC waspProjectDir
+
+  case getDbSystem decls of
+    Nothing -> noteSQLiteDoesntNeedStart
+    Just AS.App.Db.SQLite -> noteSQLiteDoesntNeedStart
+    Just AS.App.Db.PostgreSQL -> startPostgreDevDb waspProjectDir
+  where
+    getDbSystem decls = (snd <$> AS.getApp decls) >>= AS.App.db >>= AS.App.Db.system
```

Comment:
```suggestion
    getDbSystem decls = AS.getApp decls >>= AS.App.db . snd >>= AS.App.Db.system
```

Just giving you an option that Hlint gave.

Reply:
Ah I know, it gave it to me also but I didn't like it I thought it reduces readibility!